docfx.json

```
@@ -263,6 +264,7 @@
         "docs/architecture/**/**.md": "nanil",
         "docs/architecture/grpc-for-wcf-developers/**/**.md": "soshir",
         "docs/core/**/**.md": "mairaw",
+	"docs/core/run-time-config/**/**.md": "gewarren",
```

Replaced tab with spaces.

```suggestion
        "docs/core/run-time-config/**/**.md": "gewarren",
```